// try and catch

// the try statement allows to define a block of code to be tested for errors while it is being executed

// the catch statement defines a block of code to be executed, if an error occurs in the try block

// the finally statement defines a block of code to be executed regardless of the result of the try or catch blocks

// the throw statement throws an error and stops the execution of the program

try {
    // code that can throw an error
    console.log(a);
} catch {
    // code to handle the error
    console.log("variable a doesn't define ");
}
